Common/TableProducer/v0converter.cxx

```diff
@@ -23,7 +23,7 @@ struct V0Converter {
   void process(aod::V0s_001 const& v0s)
   {
     for (auto& v0 : v0s) {
-      uint8_t bitMask = static_cast<uint8_t>(255);
+      uint8_t bitMask = static_cast<uint8_t>(2); // second bit on
```

Comment:
I think the default  should be `uint8_t bitMask = static_cast<uint8_t>(1); // 1st bit on`
since this means standard v0. 2nd bit is set additionally to flag photon-only check.

Reply:
Hi @shahor02 - sorry, yes, fixed now! Thanks!